app/views/govuk_publishing_components/components/docs/government_navigation.yml

```diff
@@ -2,6 +2,8 @@ name: Government navigation
 description: Navigation placed in the header by Slimmer and used by pages which sit
   under the /government path. This is a markup only component and is not available
   for preview here. It can be passed a string to mark a link as being active.
+body: | 
+  Please note: because the markup on this component is currently tied to styles that aren't tied to this components gem, they will appear "broken". This is because they are intended to be used in the GOV.UK header as white links against a black background. You can see a styled example of this component in use [here](https://www.gov.uk/government/groups/counter-fraud-standards-and-profession). 
```

Comment:
```suggestion
  Please note: because the markup on this component is currently tied to styles that aren't present in the components gem, they will appear "broken". This is because they are intended to be used in the [header component](/component-guide/layout_header/with_left_search_and_navigation) as white links against a black background. You can see a styled example of this component in use on the ['Counter-Fraud Standards and Profession' page](https://www.gov.uk/government/groups/counter-fraud-standards-and-profession). 
```